prerelease_website/templates/index.html

```diff
@@ -38,6 +38,13 @@ <h1>Select a ROS distribution to Generate a Prerelease Command</h1>
           Lunar Loggerhead
         </a>
       </div>
+      <div class="col-sm-4 melodic-column text-center">
```

Comment:
I think, unfortunately, you need to update this one and all the other peers to be `col-sm-3`, since it's supposed to add up to 12 total. See:

https://getbootstrap.com/docs/3.3/examples/grid/

Reply:
Oh, I see.  I mistook the `col-sm-4` for the number of boxes across, but it is actually the portion of the length across.  Makes sense, I'll fix this.